lib/index.js

```diff
@@ -3,6 +3,7 @@ const missingPlaywrightAwait = require("./rules/missing-playwright-await");
 module.exports = {
   configs: {
     recommended: {
+      plugins: ['jest'],
```

Comment:
Let's also add `jest-playwright` to the list of plugins so users don't have to add that manually.

```suggestion
      plugins: ['jest', 'jest-playwright'],
```

This means we can also update the usage section in the readme.

```diff
{
  "extends": ["plugin:jest-playwright/recommended"],
- "plugins": ["jest-playwright"]
}
```

Reply:
Alright, should I also update the README with both configuration ways ? One with the extends recommended and one without ?

Reply:
Done :)